packages/liferay-npm-scripts/src/utils/filterGlobs.js

```diff
@@ -19,9 +19,11 @@ function filterGlobs(globs, ...extensions) {
 		}
 	});
 
-	return globs.filter(glob => {
-		return extensions.some(extension => glob.endsWith(extension));
-	});
+	return extensions.length
+		? globs.filter(glob => {
+				return extensions.some(extension => glob.endsWith(extension));
+		  })
+		: globs;
```

Comment:
This tweak allows me to call `getPaths` without any extensions, so that I can grab all the bad config files regardless of extension.